### DISCUSSION TITLE
Does "Send Password Reset" work? How does it work?

### DISCUSSION THREAD

**Discussion by roand0617** [hrfee/jfa-go#182] [Q&A]
Hello, first of all, Merry Christmas everyone

I wanted to ask if the "Send Password Reset" option works since I ran a test and nothing happened ...

![image](https://user-images.githubusercontent.com/95441353/147369240-562edb08-1b40-499e-a89a-1f00d6951e69.png)

Thanks

**Comment by hrfee** [comment]
It works for me, but could you open the javascript console (F12 > Console) and try this again? Look for any error messages, they should be in red. Also might be useful to check the jfa-go logs as well for anything.

**Comment by roand0617** [comment]
Hello, thanks for responding so soon

Before falling into beginner errors, could you tell me what the exact function it does? As I understand it, it sends the email or notice to the user with a new login PIN, but if that is, nothing just happens, I have my SMTP server running.

**Reply by hrfee** [reply]
Once you click it, a loading symbol should appear in the button, and it'll send a link to the selected users. After a couple seconds a box like this should appear:
![box](https://user-images.githubusercontent.com/11923517/147369604-05c142ab-413f-4d78-97a7-16971b7902ad.jpg)

**Comment by hrfee** [comment]
Ah, just realized why it doesn't work. You need to enable "Use reset link instead of PIN" in Settings > Password Resets. The button should be hidden when it's disabled.

**Comment by roand0617** [comment]
Perfect!

I did not have the option you mention activated. Thanks for your help. :)
